driver/main_args.ml

```diff
@@ -64,6 +64,12 @@ let mk_config f =
   "-config", Arg.Unit f, " Print configuration values and exit"
 ;;
 
+let mk_config_var f =
+  "-config-var", Arg.String f,
+  " Print the value of a configuration variable, a newline, and exit \
+    (print nothing and exit with error value if the variable does not exist)"
```

Comment:
Could you please cut this message with `\n\` and not make the compiler's help output look like shit (any more than it already does)?

Reply:
@damiendoligez could I implement an `Arg.wrap : ~line_length:int -> specs -> specs` function instead of manually doing the job of choosing the right cut place depending on the option name and alignment choice?

Reply:
Sure, go ahead.

Reply:
In the end I did the simple thing of adding turning the end-of-line `\` into `\n     \`.